monitoring/docker-compose.yaml

```diff
@@ -0,0 +1,104 @@
+## Docker Compose File for monitoring Kafka messages from USDF
+
+services:
+  # Grafana dashboard for monitoring Kafka data via Prometheus
+  grafana:
```

Comment:
As we are going to register the metrics in Vande we no longer need to deploy our own grafana